ipn/ipnlocal/c2n.go

```diff
@@ -48,6 +48,7 @@ var c2nHandlers = map[methodAndPath]c2nHandler{
 	req("/debug/metrics"):           handleC2NDebugMetrics,
 	req("/debug/component-logging"): handleC2NDebugComponentLogging,
 	req("/debug/logheap"):           handleC2NDebugLogHeap,
+	req("/debug/logallocs"):         handleC2NDebugLogAllocs,
```

Comment:
The "log" here is weird. I imagine you're copying "logheap" but that is weird due to history: it predated c2n so the only way we could get pprof data out of the iOS app was to log it and pick it up later async.

Why don't we go with:

```go
	req("/debug/logheap"):           handleC2NDebugPprofHeap,
	req("/debug/pprof/heap"):         handleC2NDebugPprofHeap,
	req("/debug/pprof/allocs"):         handleC2NDebugPprofAllocs,
```

(Moving to /pprof/ but keeping the legacy path too)

Reply:
Done (parametrically)